jenkinsbuilder/cccp_index_reader.py

```diff
@@ -39,7 +39,7 @@ def projectify(new_project,appid,jobid,giturl,gitpath,gitbranch,targetfile,depen
 
 def main(indexdlocation):
 
-    yamlfile = "./index.yml"
+    yamlfile = "/".join(indexdlocation.split("/")[:-1])+"/index.yml"
```

Comment:
don't do such string manipulation for paths.

Use ``os.path`` module for it: ``os.path.dirname(__file__)``, ``os.path.join``, etc